worlds/osrs/__init__.py

```diff
@@ -127,6 +130,15 @@ def interpret_slot_data(self, slot_data: typing.Dict[str, typing.Any]) -> None:
             starting_entrance.access_rule = lambda state: state.has(self.starting_area_item, self.player)
             starting_entrance.connect(self.region_name_to_data[starting_area_region])
 
+    def pre_fill(self) -> None:
+        all_state = self.multiworld.get_all_state(False)
+        for location in self.multiworld.get_locations(self.player):
+            # if not location.can_reach(all_state):
+            #     options = self.options
+            #     print(f"{location} can't be reached even with every progression item.\n{all_state.prog_items[self.player]}")
+            assert location.can_reach(
+                all_state), f"{location} can't be reached even with every progression item.\n{all_state.prog_items[self.player]}"
+
```

Comment:
This is a nice debugging tool, but I don't actually think it should be in main :D